### ISSUE TITLE
about "encoder pretrained model" and "final pretrained model" 

### ISSUE THREAD

**Issue by liminn** [xiaoyufenfei/LEDNet#19]
Thanks for your great work! 
I read your code carefully, my question are as follows:
- Can you upload the pretrained model for the encoder? This will make it easier for us to reproduce your results.
- Can you explain your hyper parameters(like initial learning rate) when pretrain the encoder?There is no explanation for these contents in the paper.
- Can you upload the pretrained model for the final model? The link in README.md is invalid.

Thanks for your work again!
